<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>phaseDialog</class>
 <widget class="QDialog" name="phaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="locale">
   <locale language="English" country="Australia"/>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>29</y>
     <width>381</width>
     <height>138</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="orderLabel">
      <property name="text">
       <string>Dislocation &amp;order</string>
      </property>
      <property name="buddy">
       <cstring>orderSpinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="xLabel">
      <property name="text">
       <string>Factor in &amp;X direction</string>
      </property>
      <property name="buddy">
       <cstring>xSpinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="yLabel">
      <property name="text">
       <string>Factor in &amp;Y direction</string>
      </property>
      <property name="buddy">
       <cstring>ySpinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="absCheckBox">
      <property name="text">
       <string>Use &amp;absolute value of phase</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="orderSpinBox">
      <property name="maximum">
       <number>2147483647</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="xSpinBox"/>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="ySpinBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>&amp;Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>absCheckBox</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>phaseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>phaseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
